config/manager/manager.yaml

```diff
@@ -57,12 +58,7 @@ spec:
             port: 8081
           initialDelaySeconds: 5
           periodSeconds: 10
-        # TODO(user): Configure the resources accordingly based on the project requirements.
-        # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
         resources:
-          limits:
-            cpu: 500m
-            memory: 128Mi
```

Comment:
Note: we have conformance tests that verify component pods don't specify limits, and don't get assigned the BestEffort quality of service. Removing the limits resulted in the Burstable QoS being assigned to these pods.